rust/common-runtime/src/bin/runtime.rs

```diff
@@ -10,26 +10,41 @@ pub fn main() {
 #[cfg(not(target_arch = "wasm32"))]
 #[tokio::main]
 pub async fn main() -> Result<(), common_runtime::CommonRuntimeError> {
+    use clap::Parser;
     use common_runtime::serve;
     use std::net::SocketAddr;
     use tracing_subscriber::{EnvFilter, FmtSubscriber};
 
+    #[derive(clap::Parser)]
+    #[command(version, about, long_about = None)]
+    struct Cli {
+        /// Set runtime server to listen on provided port.
+        #[arg(short, long, default_value_t = 8081)]
+        port: u16,
+
+        /// URL to the build server.
+        #[arg(short, long)]
+        builder_address: Option<String>,
```

Comment:
Can we make this a `Uri` instead of `String`?

Reply:
Indeed that works, must be `url::Url` that doesn't like being parsed here IIRC, updating